frontend/public/kubevirt/components/cluster/cluster-overview.jsx

```diff
@@ -137,39 +179,45 @@ export class ClusterOverview extends React.Component {
       `${k8sBasePath}/apis/subresources.${VirtualMachineModel.apiGroup}/${VirtualMachineModel.apiVersion}/healthz`,
       result => this.onFetch('kubevirtHealth', result)
     );
-    fetchPrometheusQuery(CEPH_STATUS_QUERY, result => this.onFetch('cephHealth', result));
   }
 
   componentDidMount() {
     this._isMounted = true;
 
-    fetchPrometheusQuery(CONSUMERS_CPU_QUERY, result => this.onFetch('workloadCpuResults', result));
-    fetchPrometheusQuery(CONSUMERS_MEMORY_QUERY, result => this.onFetch('workloadMemoryResults', result));
-    fetchPrometheusQuery(CONSUMERS_STORAGE_QUERY, 'workloadStorageResults');
-    fetchPrometheusQuery(CONSUMERS_NETWORK_QUERY, 'workloadNetworkResults');
+    this.fetchHealth();
 
-    fetchPrometheusQuery(NODE_CONSUMERS_MEMORY_QUERY, result => this.onFetch('infraMemoryResults', result));
-    fetchPrometheusQuery(NODE_CONSUMERS_CPU_QUERY, result => this.onFetch('infraCpuResults', result));
+    if (getPrometheusBaseURL()){
+      fetchPrometheusQuery(CEPH_STATUS_QUERY, result => this.onFetch(PROM_RESULT_CONSTANTS.cephHealth, result));
+      fetchPrometheusQuery(CONSUMERS_CPU_QUERY, result => this.onFetch(PROM_RESULT_CONSTANTS.workloadCpuResults, result));
+      fetchPrometheusQuery(CONSUMERS_MEMORY_QUERY, result => this.onFetch(PROM_RESULT_CONSTANTS.workloadMemoryResults, result));
+      fetchPrometheusQuery(CONSUMERS_STORAGE_QUERY, result => this.onFetch(PROM_RESULT_CONSTANTS.workloadStorageResults, result));
+      fetchPrometheusQuery(CONSUMERS_NETWORK_QUERY, result => this.onFetch(PROM_RESULT_CONSTANTS.workloadNetworkResults, result));
 
-    fetchPrometheusQuery(NODE_CONSUMERS_STORAGE_QUERY, 'infraStorageResults');
```

Comment:
how did this work before (#361) ? This does not seem to not call onFetch when the query finishes. + other few queries as well.

Reply:
@rawagner

Reply:
that is error in rebasing. This PR added those queries https://github.com/kubevirt/web-ui/pull/351 but after merging I didnt properly rebased #361. Thanks for catching it!